client/app/services/retire-service-modal/retire-service-modal.html

```diff
@@ -31,6 +31,7 @@ <h4 translate>Affected Services</h4>
     reset-button="vm.resetModal"
     on-reset="vm.reset($event)"
     on-cancel="vm.cancel()"
-    on-save="vm.save()">
+    on-save="vm.save()"
+    is-disabled="vm.isDisabled()">
```

Comment:
This could be made far simpler by:
`is-disabled="vm.modalData.date"`

I checked it out, works just fine